capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -58,6 +60,11 @@
           search_sql_statement
         )
       ) {
+        String new_search_word = "%" + search_word + "%";
+        for (int i = 1 ; i <= 4 ; i ++) {
```

Comment:
why 4?

Reply:
Should be formatted as: for (int i = 1; i <= 4; i++)

Reply:
Fixed the spaces. It is 4 because we have 4 placeholders in the SQL query ( represented as ?), so I need to fill out those.
So instead of repetitively writing queryStatement.setString(1, new_search_word) , queryStatement.setString(2, new_search_word) .... , I decided to use a for loop for that.